### PR TITLE
[new release] csv, csvtool and csv-lwt (2.4)

### PR DESCRIPTION
**Pull request by Chris00**
A pure OCaml library to read and write CSV files

- Project page: <a href="https://github.com/Chris00/ocaml-csv">https://github.com/Chris00/ocaml-csv</a>
- Documentation: <a href="https://Chris00.github.io/ocaml-csv/doc">https://Chris00.github.io/ocaml-csv/doc</a>

##### CHANGES:

- Fixes the build on Windows with CRLF checkout (@dra27).
- Allows use of the repo in a Duniverse (@dra27).

### CODE REVIEW

**Comment by Chris00** [comment]
Fedora issue unrelated to those packages.

File: packages/csv-lwt/csv-lwt.2.4/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+bug-reports: "https://github.com/Chris00/ocaml-csv/issues"
+doc: "https://Chris00.github.io/ocaml-csv/doc"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "csv"
```
Comment (kit-ty-kate) [inline packages/csv-lwt/csv-lwt.2.4/opam]:
```suggestion
  "csv" {= version}
```
Are you sure this dependency does not requires any specific version of `csv`?

Comment (Chris00) [reply packages/csv-lwt/csv-lwt.2.4/opam]: Thanks.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**